### PR TITLE
Allow setting blackhole-like routes

### PR DESCRIPTION
**Pull request by user318**
This pull request fixes some typos and allows using blackhole-like routes ([https://bugs.gentoo.org/637394](url))

### CODE REVIEW

**Comment by matoro** [comment]
Thanks for sending this, can you modify to our commit format:  include a signoff (`git commit --signoff`) and tag the bug like this:  `Bug: https://bugs.gentoo.org/637394` or `Closes: https://bugs.gentoo.org/637394` if it will complete the task.

**Comment by user318** [comment]
OK. I've done it and it seems to be updated here too. BTW, for commit messages I was guided by STYLE file, so it might be good idea to update it.

And if it is merged, the documentation probably needs to be updated too. I'll check it later.

**Comment by user318** [comment]
Dropped one erroneous commit that "fixed" what I believed were typos, but it is valid syntax.

**Review by matoro — APPROVED** [review]

**Comment by chutz** [comment]
This looks good to me, but please include the certificate of origin `Signed-off-by` line with your name/email as described at https://devmanual.gentoo.org/general-concepts/copyright-policy/index.html#certificate-of-origin

**Comment by matoro** [comment]
> This looks good to me, but please include the certificate of origin `Signed-off-by` line with your name/email as described at https://devmanual.gentoo.org/general-concepts/copyright-policy/index.html#certificate-of-origin

It is included already?

File: net/iproute2.sh [hunk 1]
```diff
@@ -280,13 +288,25 @@ _add_route()
 		cmd="${cmd} metric ${metric}"
 	fi
 
+	# Process dev vs nodev routes
+	# Positional parameters are used for correct array handling
+	if [[ -n ${rtype} ]]; then
```
Comment (CyberTailor) [inline net/iproute2.sh:293]: This is a bashism, single brackets should be used instead.

Comment (thesamesam) [reply net/iproute2.sh:293]: Fixed in 97a79cfd068a6cc07e9feed0ac5b71f64347ae4a, thanks.

**Pull request closed without merging**